index.html

```diff
@@ -68,219 +63,246 @@ <h1>
+          </p>
+          <a class="connect" href="#">LET'S CONNECT</a>
+          <nav class="icon-body">
+            <ul class="social-icons">
+              <li>
+                <a href="https://github.com/Goodluck5956"
+                  ><i class="fab fa-github"></i
+                ></a>
+              </li>
+              <li>
+                <a href="www.linkedin.com/in/lucky-james-osuagwu-a8677b171"
+                  ><i class="fab fa-linkedin-in"></i
+                ></a>
+              </li>
+              <li>
+                <a href=""><i class="fab fa-angellist"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fab fa-twitter"></i></a>
+              </li>
+              <li>
+                <a href=""><i class="fab fa-medium-m"></i></a>
+              </li>
+            </ul>
+          </nav>
+          <form action="#">
+            <button class="btn-body" type="submit" name="submit" id="submit">
+              get my resume
+            </button>
+          </form>
+        </div>
+        <div class="flex-item2 flex-item">
+          <ul class="language">
+            <li class="dropdowns">languages</li>
+            <li><img src="./templates/Union1.png" alt="dropdown-icon" /></li>
           </ul>
-          <ul class="html">
-            <li><img src="./templates/Ellipse6(1).svg" alt="html-logo" /></li>
-            <li>html</li>
+          <nav class="program-languages">
+            <ul class="script">
+              <li><img src="./templates/Ellipse5.svg" alt="js-logo" /></li>
+              <li>JavaScript</li>
+            </ul>
+            <ul class="html">
+              <li><img src="./templates/Ellipse6(1).svg" alt="html-logo" /></li>
+              <li>html</li>
+            </ul>
+            <ul class="css">
+              <li><img src="./templates/Ellipse7(3).svg" alt="css-logo" /></li>
+              <li>css</li>
+            </ul>
+          </nav>
+          <ul class="framework">
+            <li class="dropdowns">frameworks</li>
+            <li><img src="./templates/Union.png" alt="dropdown-icon" /></li>
           </ul>
-          <ul class="css">
-            <li><img src="./templates/Ellipse7(3).svg" alt="css-logo" /></li>
-            <li>css</li>
+          <ul class="skill">
+            <li class="dropdowns">skills</li>
+            <li><img src="./templates/Union.png" alt="dropdown-icon" /></li>
           </ul>
-        </nav>
-        <ul class="framework">
-          <li class="downlinks">frameworks</li>
-          <li><img src="./templates/Union.png" alt="dropdown-icon" /></li>
-        </ul>
-        <ul class="skill">
-          <li class="downlinks">skills</li>
-          <li><img src="./templates/Union.png" alt="dropdown-icon" /></li>
-        </ul>
+        </div>
       </section>
     </main>
     <footer class="contact">
       <section class="contact-me">
-        <h2>Contact me</h2>
+        <h3>Contact me</h3>
```

Comment:
- [ ] For the contact section, the design specs given occupy around a third of the available width. Your design occupies more than two-thirds.
Your design
![Screenshot from 2021-08-28 20-19-59](https://user-images.githubusercontent.com/65501535/131225765-8daa3592-f826-42c5-ac27-aec01c14866f.png)


Original
![Screenshot from 2021-08-28 20-19-20](https://user-images.githubusercontent.com/65501535/131225749-51f01116-4fa7-420f-9a0f-b7976fd99339.png)